src/frontend/Pretty_printing.ml

```diff
@@ -11,6 +11,38 @@ let indent_num = ref 1
 let begin_indent _ = indent_num := 1 + !indent_num
 let exit_indent _ = indent_num := -1 + !indent_num
 let tabs () = String.make (2 * !indent_num) ' '
```

Comment:
Could you delete lines 10-13, as they are dead code?

Reply:
I believe this has been addressed in 75cf2c8